### PR TITLE
Configuration option strictRules.disallowedLooseComparsion

### PR DESCRIPTION
**Pull request by MartinMystikJonas**

### CODE REVIEW

File: rules.neon [hunk 1]
```diff
@@ -16,6 +16,7 @@ parameters:
 		illegalConstructorMethodCall: %featureToggles.bleedingEdge%
 	strictRules:
 		allRules: true
+		disallowedLooseComparsion: [%strictRules.allRules%, %featureToggles.bleedingEdge%]
```
Comment (ondrejmirtes) [inline rules.neon]: Typo

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: rules.neon (hunk 1 above)
Comment (MartinMystikJonas) [reply rules.neon]: OMG I was writing this word wrong my whole life :-D Fixed

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]
And please update the README too 😊

**Comment by MartinMystikJonas** [comment]
Updated

**Comment by ondrejmirtes** [comment]
Thank you!

**Pull request merged**